### ISSUE TITLE
COntent acknowledgement and page with RA names etc/adding testimonials

### ISSUE THREAD

**Issue by shrabasteebanerjee** [tilburgsciencehub/website#643]

**Comment by hannesdatta** [comment]
We have this here:

https://tilburgsciencehub.com/contributors/

It takes the author names from the meta data of articles.

@lachlandeer - which next steps do you propose?

**Comment by shrabasteebanerjee** [comment]
1. There should be a way to "click" these tabs and be led to the set of
articles written by a given RA/researcher.
2. We should have a "current" and "past" section
3. For the "past" RAs, we should include some testimonials (I know these
appear on our onboarding guide here:
https://github.com/tilburgsciencehub/onboarding/wiki/Alumni ) but they
should be featured on the main website too!

**Comment by shrabasteebanerjee** [comment]
PS: Also not all RAs appear on the current contributors page, e.g Paulina,
Gaston, Thierry..

**Comment by hannesdatta** [comment]
Good idea, @shrabasteebanerjee.

__Deliverables:__
- Draft deliverables

A few notes:
- get inspiration from other hugo installations about author pages 
- find out how to 'link' articles to authors; right now, we just use full names (but that doesn't allow us to tag them very well and link to author pages w/ extra details)
- define what should be on an author page
- work on the layout of these pages (i.e., overview of contributors, but also an authors' articles page)

Anything to add?

**Comment by lachlandeer** [comment]
One reason to link contributions is to give the author something tangible to say "hey, here's what I did while I worked with TSH" and it gives them a more recognition (something I've been thinking about lately, in terms of our contributors but also a biblio on each block where we've clearly borrowed stuff from elsewhere). 

Should also find a way to have multiple authors on a post as I can forsee this becoming something we'll want soon. 

On design: Something similar to the wowchemy academic wouldn't be a bad place to start, see for example [this](https://research-group.netlify.app/people/)

Where we could have Core Team, Current Contributors, Past Contributors. I think if one was to click into each person's profile one gets there publications / posts
